Resources/config/yui.yml

```diff
@@ -11,13 +11,34 @@ system:
                         - 'cof-universaldiscoverycreateview'
                     dependencyOf: ['ez-universaldiscoveryview']
                     path: %ezcontentonthefly.public_dir%/js/views/plugins/cof-createcontent-universaldiscoveryplugin.js
+                cof-selectcontenttypeplugin:
+                    requires:
+                        - 'ez-contentcreateplugin'
```

Comment:
Nitpick: You can write `requires: ['ez-contentcreateplugin']`. There's no need to keep it in spearate line